com.unity.netcode.gameobjects/Tests/Runtime/NetworkShowHideTests.cs

```diff
@@ -213,5 +213,44 @@ public IEnumerator NetworkShowHideTest()
                 yield return CheckVisible(true);
             }
         }
+
+        [UnityTest]
+        public IEnumerator NetworkShowHideQuickTest()
+        {
+            m_ClientId0 = m_ClientNetworkManagers[0].LocalClientId;
+
+            var spawnedObject1 = UnityEngine.Object.Instantiate(m_PrefabToSpawn);
+            var spawnedObject2 = UnityEngine.Object.Instantiate(m_PrefabToSpawn);
+            var spawnedObject3 = UnityEngine.Object.Instantiate(m_PrefabToSpawn);
+            m_NetSpawnedObject1 = spawnedObject1.GetComponent<NetworkObject>();
+            m_NetSpawnedObject2 = spawnedObject2.GetComponent<NetworkObject>();
+            m_NetSpawnedObject3 = spawnedObject3.GetComponent<NetworkObject>();
+            m_NetSpawnedObject1.NetworkManagerOwner = m_ServerNetworkManager;
+            m_NetSpawnedObject2.NetworkManagerOwner = m_ServerNetworkManager;
+            m_NetSpawnedObject3.NetworkManagerOwner = m_ServerNetworkManager;
+            m_NetSpawnedObject1.Spawn();
+            m_NetSpawnedObject2.Spawn();
+            m_NetSpawnedObject3.Spawn();
+
+            for (int mode = 0; mode < 2; mode++)
+            {
+                // get the NetworkObject on a client instance
+                yield return RefreshNetworkObjects();
+
+                // check object start visible
+                yield return CheckVisible(true);
+
+                // hide and show them on one client, during the same frame
+                Show(mode == 0, false);
+                Show(mode == 0, true);
+
+                yield return new WaitForSeconds(0.5f);
```

Comment:
perhaps waitforticks (network) instead of waitforseconds?

Reply:
Also, done using ticks :-)